payload/reggae/backend/binary.d

```diff
@@ -222,7 +221,7 @@ private:
     //Checks dependencies listed in the .dep file created by the compiler
     bool checkDeps(Target target, in string depFileName) @trusted {
 
-        auto file = File(depFileName);
+        auto file = File(depFileName, "r");
         auto lines = file.byLine.map!(a => a.to!string);
```

Comment:
`byLine` fails badly for Windows text files with CRLF line terminators if opened in (default) `rb` binary mode.

Reply:
Good to know! Could you please add a comment to that effect in the code itself? Thanks.

Reply:
Added to this location; there would be some more.